pdf_parser.py

```diff
@@ -0,0 +1,40 @@
+import fitz
+from common_utils import *
+
+# PDF files to png pages parser
+class PDF_parser:
+    def __init__(self, output_folder=os.environ['FOLDER_PAGES']):
+        self.page_output_folder = output_folder
+
+        self.pdf_file_list = []
+
+        if not os.path.exists(self.page_output_folder):
+            os.makedirs(self.page_output_folder)
+
+    # load PDF, go through pages and save them as PNG
+    def pdf_to_png(self, pdf_file: str, large_size=True) -> None:
+        items = [page_file for page_file in os.listdir(self.page_output_folder) if
```

Comment:
this requires a comment.

Reply:
comment to what?

Reply:
what is the calculation on the left of items (the thing between [ and ]). It is not something that is obvious.